<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-TJKKK52O" name="com.Docker.Selenium_Docker.LoginTest" tests="4" failures="2" timestamp="2021-06-09T14:37:27 IST" time="13.979" errors="0">
  <testcase name="Login_Test" time="5.921" classname="com.Docker.Selenium_Docker.LoginTest"/>
  <system-out/>
  <testcase name="afterTest" time="0.764" classname="com.Docker.Selenium_Docker.LoginTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Test Name Failed : USER LOGIN
Failed while Validating  : Failing of Assertion
Class Failed : com.Docker.Selenium_Docker.LoginTest
 METHOD FAILED  : Login_Test
 IN Line number : 37
Assertion Fail in URL : https://demo.openmrs.org/openmrs/referenceapplication/home.page
 Failure Message  : Why are you failing expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Test Name Failed : USER LOGIN
Failed while Validating  : Failing of Assertion
Class Failed : com.Docker.Selenium_Docker.LoginTest
 METHOD FAILED  : Login_Test
 IN Line number : 37
Assertion Fail in URL : https://demo.openmrs.org/openmrs/referenceapplication/home.page
 Failure Message  : Why are you failing expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at resource.BaseTest.afterTest(BaseTest.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:617)
at org.testng.TestRunner.afterRun(TestRunner.java:847)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- afterTest -->
  <system-out/>
  <testcase name="Login_Test" time="6.602" classname="com.Docker.Selenium_Docker.LoginTest"/>
  <system-out/>
  <testcase name="afterTest" time="0.692" classname="com.Docker.Selenium_Docker.LoginTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Test Name Failed : USER LOGIN Invalid User
Failed while Validating  : Failing of Assertion
Class Failed : com.Docker.Selenium_Docker.LoginTest
 METHOD FAILED  : Login_Test
 IN Line number : 37
Assertion Fail in URL : https://demo.openmrs.org/openmrs/referenceapplication/login.page
 Failure Message  : Why are you failing expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Test Name Failed : USER LOGIN Invalid User
Failed while Validating  : Failing of Assertion
Class Failed : com.Docker.Selenium_Docker.LoginTest
 METHOD FAILED  : Login_Test
 IN Line number : 37
Assertion Fail in URL : https://demo.openmrs.org/openmrs/referenceapplication/login.page
 Failure Message  : Why are you failing expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at resource.BaseTest.afterTest(BaseTest.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:617)
at org.testng.TestRunner.afterRun(TestRunner.java:847)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- afterTest -->
  <system-out/>
</testsuite> <!-- com.Docker.Selenium_Docker.LoginTest -->
